dandischema/models.py

```diff
@@ -960,6 +1048,12 @@ class BareAsset(CommonModel):
     blobDateModified: Optional[datetime] = Field(
         None, nskey="dandi", title="Asset file modification date and time"
     )
+    access: List[AccessRequirements] = Field(
```

Comment:
we overload this one just to add `max_item=1` for the `BareAsset`? might be worth a comment

```suggestion
    # overload to restrict with max_items=1
    access: List[AccessRequirements] = Field(
```
even thought we might have not followed the same for other fields in similar situations yet (I think we need a helper to make it more obvious etc... not here though ;-))